tty/imx6ull-uart/imx6ull-uart.c

```diff
@@ -29,12 +32,71 @@
 #include <sys/stat.h>
 #include <sys/debug.h>
 #include <posix/utils.h>
+#include <fcntl.h>
+#include <syslog.h>
 
 #include <libtty.h>
 #include <libklog.h>
 
 #include <phoenix/arch/imx6ull.h>
 
+#define COL_RED    "\033[1;31m"
+#define COL_YELLOW "\033[1;33m"
+#define COL_NORMAL "\033[0m"
+
+#define LOG_TAG "imx6ull-uart"
+
+/* clang-format off */
+#define log_debug(fmt, ...) do { log_printf(LOG_DEBUG, fmt "\n", ##__VA_ARGS__); } while (0)
+#define log_info(fmt, ...)  do { log_printf(LOG_INFO, fmt "\n", ##__VA_ARGS__); } while (0)
+#define log_warn(fmt, ...)  do { log_printf(LOG_WARNING, COL_YELLOW fmt COL_NORMAL "\n", ##__VA_ARGS__); } while (0)
+#define log_error(fmt, ...) do { log_printf(LOG_ERR, COL_RED fmt COL_NORMAL "\n", ##__VA_ARGS__); } while (0)
+/* clang-format on */
```

Comment:
Do we need this? I don't see these used... Only log_warn is used. Imho such logs should be for debug only, not enabled by default (so disabled under #if NDEBUG)

Reply:
No, I can use `printf` directly if you don't like these defines. Currently I use `log_warn` to print UART errors which is useful for checking problems on DCU. These errors are not printed by default (option `-e`).

Reply:
Ok, perhaps might be useful for the future debugging, developement